APE0.rst

```diff
@@ -0,0 +1,360 @@
+  coordination meeting in the last two years,
+* has a named role in the project and has regularly participated in Astropy
+  telecons (e.g., co-working telecon, spectroscopy sprint, Astropy Learn
+  telecon) in the last two years,
+* has commit rights to at least one repository in the astropy GitHub
+  organization and has actively used those commit rights in the last two years.
+
+Add New Voting Members
+^^^^^^^^^^^^^^^^^^^^^^
+Anyone can apply to become a Voting Member by providing evidence of meeting the
+requirements laid out in the Membership section above. Both self-nominations and
+nominations by others are allowed. The Voting Members are expected to make their
+decisions based on the candidate’s adherence to the membership criteria, above.
+The name of the nominee will be known to Voting Members but will not be shared
+outside Voting Members and the Coordination Committee unless the nominee becomes
+a Voting Member.
+
+The procedure for voting to add new Voting Members is:
+
+* The Coordination Committee and Ombudsperson receive each nomination, check
+  that it is factually accurate, that the nominated person accepts the
+  nomination, and that their record of community activity adheres to the Code of
+  Conduct.
+* Once a nomination is accepted by the Coordination Committee, the Coordination
+  Committee will initiate a vote among active Voting Members. The voting will be
+  open for at least two weeks and will conclude within four weeks of the
+  nomination being accepted. To successfully gain an appointment as a Voting
+  Member, the nominee must receive at least four positive votes, as long as that
+  constitutes a majority of those voting.
+* The candidate will be informed promptly at the close of voting by a
+  Coordination Committee member. If the vote is not affirmative, the
+  Coordination Committee will provide feedback to the nominee.
+
+Term and Active/Inactive Status
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Voting Members have no term or term limits. Voting Members who have stopped
+contributing are encouraged to declare themselves as "inactive.” Those who have
+not made any significant contribution for a long period may be asked to move
+themselves to the “inactive” category by the Coordination Committee. If no
+response is received, the Coordination Committee may automatically change a
+Voting Member’s status to inactive. To record and honor their contributions,
+inactive Voting Members will continue to be listed. Inactive Voting Members are
+not able to participate in votes.
+
+Removing Voting Members
+^^^^^^^^^^^^^^^^^^^^^^^
+In exceptional circumstances, it may be necessary to remove someone from the
+Voting Members against their will. A vote must be held to remove a Voting
+Member. Such a vote is triggered by a motion made by an active Voting Member,
+which must be seconded by an additional Voting Member. The vote must conclude
+no more than four weeks after the motion is seconded. Removal requires approval
```

Comment:
Similarly to the above, I think four weeks is a very tight deadline. It's probably OK here because this is going to be a rare case, but I don't see urgency. If someone really wrecks havoc, the coco can still remove github privileges related to that account immediately and then there is little harm if that person is still a voting member till the next vote is called.

Reply:
I think the intent of all deadlines is that we don't want it to hang over someone's head forever.  It's like how the US Constitution specifies a "fair and speedy trial".  But to your point it does *not* say "within 4 weeks", so I guess these could be changed to something more nebulous like "as soon as reasonably possible"?

Reply:
I would prefer to keep concrete times in rather than the vaguenesses. Whether that is 4 weeks or 3 months, I don't mind. 
But "speedy", "regular", etc is very vague as subject to subjective interpretation which I think is avoidable.

Reply:
Can we combine the votes for CoCo with those for new members, etc.? Once per year may be too infrequent, but if there were also a "midterms", would that be good enough?  A month or two after each release might be a sensible time.

Reply:
I like @mhvk's suggestion here, but one of the intents in this document was to *not* lock in all the details of the election right now.  For example, the exact platform for the voting and how hard it is might influence the answer to this. So I see the intent is setting bounds and guidelines that are acceptable without specifying all the details such that the APE has to be amended because, say, a certain voting platform suddenly goes out-of-business. So I think it makes sense to change the text to *allow* less frequent voting, but not *require* it or specify a specific cadence too particularly.

That said, on the removal process, the WG was thinking that if there are votes to remove people many times a year something has gone terribly wrong already! So unlike adding new members where it makes sense to roll them together because that may be a relatively frequent occurrence if the Project is healthy, the hope is that removal happens very rarely if ever, and hence is worth a specific vote because it should be treated as a serious occasion.  So I think removals ideally wouldn't be rolled into other votes (whereas adding people or additional votes we haven't covered could indeed be rolled into a twice-a-year deadline).

So that makes me want to stick with either 4 weeks or 3 months for this clause, assuming the "voting members" bit is updated to allow 6-month gaps?

Reply:
Good point that removals should not be lumped together with other votes. A bit more time than 4 weeks still seems sensible.

Reply:
I'm OK with changing "4 weeks" to "3 months". Nothing stops us to call that vote faster if possible, but if our preferred voting provider just went out of business, two Coco members had babies and third one dies in a plane crash (or, to cite a much less likely nightmare scenario, a global pandemic wrecks havok on everyones schedule), then we can stretch it to 3 months.

Reply:
Changed to 3 months